### ISSUE TITLE
No Volume slider in Chrome (Fedora with Wayland)

### ISSUE THREAD

**Issue by frederik-sevenval** [jwplayer/jwplayer#1660]
### Expected Behavior

A volume slider is shown (works on Firefox on the same OS and on Chrome on other OSs).
(Click to toggle mute is working)
The html nodes don't get added (they do on the other browser/OS combos).

### Actual Behavior

Chrome (Version 55.0.2883.87 (64-bit)) on Fedora (with wayland) dosn't display a volume slider.

### Steps to reproduce

1. setup JW player
2. open in Chrome
3. hover volume button

### Environment
- *Fedora 25 with Wayland*
- *Chrome Version 55.0.2883.87 (64-bit)*
- *JW Player version 7.6.8*

**Comment by sevenfred** [comment]
I changed the file format (webm to mp4) and now it works,
so apparently this only occurs when chrome uses the flash fallback to render webm videos (or maybe other formats aswell).

**Comment by johnBartos** [comment]
@sevenfred We're in the process of improving a lot of our skins in 7.9 so hopefully this will be fixed there. However, we don't have any developers running Fedora - your best bet will be to submit a PR which fixes the issue and let us verify it on our end. I'm happy to help you through the process

**Comment by robwalch** [comment]
Hmm, not sure why the DOM nodes wouldn't be added. Can you open the debugger in Fedora and have a look at what happens here https://github.com/jwplayer/jwplayer/blob/feaec89e54aaecfe9a58239beba68e94f7e5a393/src/js/view/components/volumetooltip.js#L14-L14 and here https://github.com/jwplayer/jwplayer/blob/6ca29b83dd9e7c4d7ab6514a7efbcdbe28073ecc/src/js/view/components/timeslider.js#L11-L11?

**Comment by frederik-sevenval** [comment]
Yeah, no... ^^
Firefox didn't show the issue at all.

I just tried again in chrome and I think it worked (, but I don't have the original video file anymore).
I think in the end it was my fault, because I used chrome dev tools in mobile emulating mode... or something like that. (That reproduces the result successfully, but I'm only 60% sure it really was the reason.)

Closing this, because in the current setup (different video source, new chrome version, etc.) I can't reproduce it as a bug, only as a feature (no volume slider in mobile device emulation).
